frontend/packages/dev-console/src/components/import/__tests__/import-validation-utils.spec.ts

```diff
@@ -46,12 +46,14 @@ describe('ValidationUtils', () => {
 
   describe('createComponentName', () => {
     const invalidConvertedtoValidNamePair: { [key: string]: string } = {
-      '0name': 'ocp-0name',
-      '-name': 'ocp--name',
-      'name-': 'ocp-name',
-      'invalid&name': 'ocp-invalidname',
-      'invalid name': 'ocp-invalidname',
-      'invalid-Name': 'ocp-invalid-name',
+      '0name': 'ocp-0-name',
+      Name: 'name',
+      '-name': 'name',
```

Comment:
Based on the previouse comment, you could add the following test data to this list as well.
 `-2name': 'ocp-2-name`,